### DISCUSSION TITLE
Problems when installing the last version from scratch

### DISCUSSION THREAD

**Discussion by CommanderPaladin** [Security-Onion-Solutions/securityonion#7580] [Unsupported Versions] [answered]
Hello kind people,
I'm trying to install the latest STANDALONE version of Security Onion on a new clear Ubuntu 20.04 system.

I have encountered the following error during the installation. I'm struggling for a while but maybe you can give me a helping hand into this. I really appreciate any help. 

This is the sosetup.log file

https://pastebin.com/0pWDw72y

Thanks you kindly.

**Comment by dougburks** [comment, marked as answer]
Looking at the log, it appears you may have ran `so-setup iso`. This is only for folks using our Security Onion ISO image.

If you're using Ubuntu 20.04, then you'll instead want to run `so-setup-network` as shown at:
https://docs.securityonion.net/en/2.3/installation.html#installation-on-ubuntu-or-centos

**Reply by CommanderPaladin** [reply]
I see, it was my bad. Thanks you so much! Now everything works perfectly!

**Reply by dougburks** [reply]
We've added some logic to help prevent this in the future:
https://github.com/Security-Onion-Solutions/securityonion/issues/7601
